packages/visx-axis/src/types.ts

```diff
@@ -75,6 +80,8 @@ export type CommonProps<Scale extends AxisScale> = {
   strokeWidth?: number | string;
   /** The pattern of dashes in the stroke. */
   strokeDasharray?: string;
+  /** The pixel value for the width of the ticks. */
```

Comment:
Oops one stale comment from previous implementation

```suggestion
  /** Props to be applied to individual tick lines. */
```

Reply:
Done ✅